src/BigwigDataFetcher.js

```diff
@@ -311,6 +328,9 @@ const BBIDataFetcher = function BBIDataFetcher(HGC, ...args) {
           tile.denseDataExtrema = dde;
           tile.minNonZero = dde.minNonZeroInTile;
           tile.maxNonZero = dde.maxNonZeroInTile;
+
+          // console.log(`tile ${JSON.stringify(tile.tilePos)}: ${tile.min_value} - ${tile.max_value}`);
```

Comment:
Remove these commented out log statements.

Reply:
I removed these and other commented log statements.